websites/B/BreathFM/presence.ts

```diff
@@ -0,0 +1,69 @@
+const presence = new Presence({
+		clientId: "846282107462352927",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	let presenceData: PresenceData;
+	if (
+		"mediaSession" in navigator &&
+		navigator.mediaSession.metadata !== null &&
+		!document.querySelector<HTMLAudioElement>("audio")?.paused
+	) {
+		presenceData = {
+			largeImageKey: navigator.mediaSession.metadata.artwork[0].src,
+			details: navigator.mediaSession.metadata.title,
+			state: navigator.mediaSession.metadata.artist,
+			buttons: [
+				{
+					label: "Listen",
+					url: location.href,
+				},
+			],
+		};
+	} else {
+		presenceData = {
+			startTimestamp: browsingTimestamp,
+			largeImageKey: "https://i.imgur.com/C8eRVDU.jpg",
+			buttons: [
+				{
+					label: "View Site",
+					url: location.href,
+				},
+			],
+		};
```

Comment:
Buttons that could lead to main pages are [not allowed](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md#presencets)
```suggestion
```